crates/deadpool/src/managed/pool.rs

```diff
@@ -416,8 +416,8 @@ pub(crate) struct PoolInner<M: Manager> {
     manager: M,
     next_id: AtomicUsize,
     slots: Mutex<Slots<ObjectInner<M>>>,
-    /// Number of ['Pool'] users. A user is both a future which is waiting for an ['Object'] or one
-    /// with an ['Object'] which hasn't been returned, yet.
+    /// Number of [`Pool`] users. A user is any future that has acquired a semaphore permit,
+    /// including both those waiting for an [`Object`] and those currently holding an [`Object`].
```

Comment:
I remember rewriting this sentence over and over again. I tried incorporating the idea of "borrowing objects from the pool" but couldn't come up with a great phrasing.